x/staking/simulation/operations_test.go

```diff
@@ -88,7 +91,12 @@ func (s *SimTestSuite) SetupTest() {
 		stakingKeeper *stakingkeeper.Keeper
 	)
 
-	app, err := simtestutil.SetupWithConfiguration(testutil.AppConfig, startupCfg, &s.txConfig, &bankKeeper, &accountKeeper, &mintKeeper, &distrKeeper, &stakingKeeper)
+	cfg := depinject.Configs(
+		testutil.AppConfig,
+		depinject.Supply(sdklog.NewNopLogger()),
```

Comment:
Should we use in tests the `log.NewTestLogger(t)` instead of nop? Or don't you think it is useful?

Reply:
not sure, before this a nop logger was used iirc. Either is fine with me, wdyt? I'm in favor of using the test logger although not sure if that will pollute our test logs